ydb/core/statistics/aggregator/aggregator_impl.cpp

```diff
@@ -396,15 +396,24 @@ size_t TStatisticsAggregator::PropagatePart(const std::vector<TNodeId>& nodeIds,
 }
 
 void TStatisticsAggregator::Handle(TEvPipeCache::TEvDeliveryProblem::TPtr& ev) {
-    auto tabletId = ev->Get()->TabletId;
-    if (ShardRanges.empty()) {
+    if (!ScanTableId.PathId) {
         return;
     }
-    auto& range = ShardRanges.front();
-    if (tabletId != range.DataShardId) {
-        return;
+    auto tabletId = ev->Get()->TabletId;
+    if (IsColumnTable) {
+        if (tabletId == HiveId) {
+            Schedule(HiveRetryInterval, new TEvPrivate::TEvRequestDistribution);
+        }
+    } else {
+        if (ShardRanges.empty()) {
+            return;
+        }
+        auto& range = ShardRanges.front();
+        if (tabletId != range.DataShardId) {
+            return;
```

Comment:
Please, add more `SA_LOG_D`, especially before `return` like this. There are many such a places.